<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MedicalDbDlg</class>
 <widget class="QDialog" name="MedicalDbDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>622</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Database Entry</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTreeWidget" name="dbListView">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectItems</enum>
     </property>
     <property name="columnCount">
      <number>4</number>
     </property>
     <column>
      <property name="text">
       <string>Fracture Site</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Category Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>First Classification</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Second Classification</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Add</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="addFracPartBtn">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Fracture site</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/Medical3D/Resources/add.png</normaloff>:/Medical3D/Resources/add.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addClassNameBtn">
          <property name="text">
           <string>Category Name</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/Medical3D/Resources/add.png</normaloff>:/Medical3D/Resources/add.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addLevel1Btn">
          <property name="text">
           <string>First Classification</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/Medical3D/Resources/add.png</normaloff>:/Medical3D/Resources/add.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addLevel2Btn">
          <property name="text">
           <string>Second Classification</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/Medical3D/Resources/add.png</normaloff>:/Medical3D/Resources/add.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Delete</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="deleteItemBtn">
          <property name="text">
           <string>Delete Selected</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/Medical3D/Resources/delete.png</normaloff>:/Medical3D/Resources/delete.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>108</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okBtn">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="medical3d.qrc"/>
 </resources>
 <connections/>
</ui>
